book/src/user/mining.md

```diff
@@ -0,0 +1,167 @@
+# Mining Zcash with zebra
+
+Zebra's RPC methods support miners and mining pools. 
+
+Contents:
+
+- [Download and build Zebra](#download-and-build-zebra)
+- [Configure zebra for mining](#configure-zebra-for-mining)
+  - [Miner address](#miner-address)
+  - [RPC section](#rpc-section)
+- [Running zebra](#running-zebra)
+- [Testing the setup](#testing-the-setup)
+- [Run a mining pool](#run-a-mining-pool)
+
+## Download and build Zebra for mining
+[#download-and-build-zebra]: #download-and-build-zebra
+
+```console
+cargo install --locked --features getblocktemplate-rpcs --git https://github.com/ZcashFoundation/zebra zebrad
+```
+
+Zebra binary will be at ` ~/.cargo/bin/zebrad`.
+
+It is important to install with the `getblocktemplate-rpcs` feature so the final binary will contain mining support.
+
+## Configure zebra for mining
+[#configure-zebra-for-mining]: #configure-zebra-for-mining
+
+We need a configuration file. First, we create a file with the default settings:
+
+```console
+zebrad generate -o ~/.config/zebrad.toml
+```
+
+The above command places the generated `zebrad.toml` config file in the default preferences directory of Linux. For other OSes default locations [see here](https://docs.rs/dirs/latest/dirs/fn.preference_dir.html).
+
+Tweak the following options in order to prepare for mining.
+
+### Miner address
+[#miner-address]: #miner-address
+
+Node miner address is required. At the moment zebra only allows `p2pkh` or `p2sh` transparent addresses.
+
+```
+[mining]
+miner_address = 't3K4aLYagSSBySdrfAGGeUd5H9z5Qvz88t2'
+```
+
+### RPC section
+[#rpc-section]: #rpc-section
+
+This change is required for zebra to behave as an RPC endpoint. The standard port for RPC endpoint is `8232`.
+
+```
+[rpc]
+listen_addr = "127.0.0.1:8232"
+```
+
+## Running zebra
+[#running-zebra]: #running-zebra
+
+If the configuration file is in the default directory, then zebra will just read from it. All we need to do is to start zebra as follows:
+
+```console
+zebrad
+```
+
+You can specify the configuration file path with `-c /path/to/config.file`.
+
+Wait until zebra is in sync, you will see the sync at 100% when this happens:
+
+```console
+...
+2023-02-21T18:41:09.088931Z  INFO {zebrad="4daedbc" net="Main"}: zebrad::components::sync::progress: finished initial sync to chain tip, using gossiped blocks sync_percent=100.000% current_height=Height(1992055) network_upgrade=Nu5 remaining_sync_blocks=1 time_since_last_state_block=0s
+...
+```
+
+## Testing the setup
+[#testing-the-setup]: #testing-the-setup
+
```

Comment:
This large command output might make the docs hard to read. Can you put it in a details block? Or replace it with a smaller example?

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections